src/components/Composer.vue

```diff
@@ -817,13 +817,10 @@ export default {
 	background-position: 12px center;
 	margin-left: 4px;
 }
-</style>
-
-<style lang="scss" scoped>
```

Comment:
I dont know what happened here and how did we miss it but I didnt do it :person_fencing: